source/routing/rendering-a-template.md

```diff
@@ -1,21 +1,19 @@
-One of the most important jobs of a route handler is rendering the
+One job of a route handler is rendering the
 appropriate template to the screen.
 
-By default, a route handler will render the template into the closest
-parent with a template.
+By default, a route handler will render the template with the same name as the
+route. For example, the `post` route will render the `post.hbs` template, and
+the `posts.new` route will render `posts/new.hbs`.
 
-```app/router.js
-Router.map(function() {
-  this.route('post');
-});
-```
+Each template will be rendered into the `{{outlet}}` of its parent route's
```

Comment:
this is tricky because you can name the route `posts.new` without it being the child, kinda. would it benefit from a small router example?